AirCasting/Utils/LocationStates.swift

```diff
@@ -0,0 +1,16 @@
+// Created by Lunar on 14/07/2021.
+//
+
+import Foundation
+
+enum LocationSates {
+    case granted
+    case denied
+    
+    var bool: Bool {
```

Comment:
Please rename this variable to sth more descriptive

Reply:
I think that when I am using it .bool making it clear what I want, I don't have an idea for the better one. Could You please suggest 😇?

Reply:
if this one is here to stay i must have a descriptive name. Currently bool is pretty clear, but if/when we add another case, for example `case undetermined` than `var bool` says nothing. Maybe let's look at how it's used and adjust the name accordingly and also move this into an `extension` placed in the same file as it's being used:

PowerABView.swift
```Swift
extension LocationStates {
    var continueButtonEnabled: Bool { ... }
}
```

The above seems all right, but this usage is very specific so what I would suggest is to really move this responsibility of determining `enabled` state to where it belongs. So in this case to some `private func` inside `PowerABView`:

```Swift
struct PowerABView: View {
    ...
    private var continueButtonEnabled: Bool { ... } 
    ...
}
```

Reply:
Thanks, I will try to change it well 😄